tests/jballerina-unit-test/src/test/resources/test-src/expressions/group/group-expr.bal

```diff
@@ -0,0 +1,99 @@
+type PersonObj object {
```

Comment:
Shall we add the copyright notice as well.

Reply:
Added in ed2962f